formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -415,3 +417,67 @@ let asd2 = video test::false 10 [@JSX] [@foo];
 let div children => 1;
 
 ((fun () => div) ()) [] [@JSX];
+
+let myFun () =>
+  <>
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true anotherOptional=200>
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+    </Namespace.Foo>
+  </>;
+
+let myFun () => <> </>;
+
+let myFun () =>
+  <>
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true anotherOptional=200>
```

Comment:
They will typically render as your `div`s. It's just that for multiple attributes, they will first break as `Namespace.Foo` does above, and then if they still don't have space, they will quickly begin to break as:

```
<Namespace.Foo
  intended=true
  another=false>
  <child />
</Namespace.Foo>
```

There's just a chance that it breaks as:

```
<Namespace.Foo
  intended=true anotherOptional=200>
```

If it saves lines.